pandas/tests/scalar/test_na_scalar.py

```diff
@@ -145,7 +147,7 @@ def test_logical_and():
     assert False & NA is False
     assert NA & NA is NA
 
-    with pytest.raises(TypeError):
+    with pytest.raises(TypeError, match="unsupported operand type"):
```

Comment:
Can you take the value of ```match``` outside the ```pytest.raises```

```suggestion
    msg = "unsupported operand type"
    with pytest.raises(TypeError, match=msg):
```

Reply:
@MomIsBestFriend I might have missed a discussion on this, but we use this pattern a lot for short, single use messages.

Reply:
Oh didn't know about that, thank you for informing me @simonjayhawkins

Reply:
no problem, if we don't have a standard, is a matter of personal preference. we started adding the messages before we adopted black, so the norm was to have a separate assignment to improve the formatting. I would not be adverse to in-lining for all single use messages now as black would keep the longer messages tidy.